packages/inferno-vnode-flags/src/index.ts

```diff
@@ -17,8 +18,14 @@ const enum VNodeFlags {
 	Void = 1 << 12,
 
 	FormElement = InputElement | TextareaElement | SelectElement,
-	Element = HtmlElement | SvgElement | MediaElement | InputElement | TextareaElement | SelectElement,
+	Element = HtmlElement |
```

Comment:
It's the first time I've ever seen binary or being chained so much that they overflow. The code for handling leading type unions was pretty difficult to get right. I would punt on this. I doesn't look bad with them on the right.